### ISSUE TITLE
Bootstrap 4 template isn't consistent

### ISSUE THREAD

**Issue by shengslogar** [diglactic/laravel-breadcrumbs#17]
Missing `aria-current` and wrapping `<nav>` per spec.

```
<nav aria-label="breadcrumb">
  <ol class="breadcrumb">
    <li class="breadcrumb-item"><a href="#">Home</a></li>
    <li class="breadcrumb-item active" aria-current="page">Library</li>
  </ol>
</nav>
```

https://getbootstrap.com/docs/4.6/components/breadcrumb/

If we change this now, this could be a breaking change. May leave it for now just to avoid that.
